crates/matrix-sdk/src/encryption/mod.rs

```diff
@@ -1352,4 +1364,4 @@ mod tests {
         let after_taking_lock_second_time = client.olm_machine().await.as_ref().unwrap().clone();
         assert!(after_taking_lock_first_time.same_as(&after_taking_lock_second_time));
     }
-}
+}
```

Comment:
You removed the final newline in this file.

Reply:
Fixed [here](https://github.com/matrix-org/matrix-rust-sdk/pull/2697/commits/5b1178d3b638095c7c5c27ca6b6d07edec87d455)